source/time_tables/departure_boards.cpp

```diff
@@ -44,6 +44,28 @@ namespace pt = boost::posix_time;
 
 namespace navitia { namespace timetables {
 
+static bool is_last_stoptime(const nt::StopTime* stop_time, const nt::StopPoint* stp){
+    return stop_time->vehicle_journey
+            && stop_time->vehicle_journey->route
+            && stop_time->vehicle_journey->route->destination
```

Comment:
why do you need to check that you have a destination here? (maybe to "outsource")